doc/etc/config.docker.ini

```diff
@@ -0,0 +1,33 @@
+# Configuration for usage inside docker container
+# See example config for comments
+
+[Server]
+address                 = 127.0.0.1
+port                    = 10000
+neoport                 = 10001
+data_dir                = data
+channel_size_limit      = 2000000000
+cpu                     = -1
+unix_socket_path        = dmqnode.socket
+
+[ChannelSizeById]
+admedia = 400000000
```

Comment:
I don't think this is necessary.

Reply:
Maybe you can just paste here whole recommended config to be used for starting a single node locally?

Reply:
(or even push it to PR directly)

Reply:
Ping @gavin-norman-sociomantic

Reply:
Apart from my 2 comments, what you have looks good.